src/language/CodeInspection.js

```diff
@@ -199,6 +199,18 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Returns an array of the names of providers registered for a specific language
+     *
+     * @param {!string} languageId
+     * @return {Array.<string>} Names of registered providers.
+     */
+    function getProviderNamesForLanguage(languageId) {
+        return _.map(_providers[languageId], function (provider) {
```

Comment:
You can use simple ES5 `_providers[languageId].map()` here as it's a simple array.